setup-env/action.yml

```diff
@@ -21,7 +21,7 @@ inputs:
   clojure-version:
     description: Version of Clojure CLI to install
     required: true
-    default: '1.10.3.943'
+    default: '1.10.3.1020'
```

Comment:
Isn't the latest version 1.10.3.1040? https://clojure.org/releases/tools

Reply:
indeed. that was just what happened to be on my machine, I'll update it